vuepress/docs/next/docs/create/create.md

```diff
@@ -0,0 +1,20 @@
+---
```

Comment:
All the text for this new file and for How to use this website is temporary. It is just to give you a feel for how it will feel and what might appear on the page. Don't waste your time reviewing it!